spec/flipper/adapters/http_spec.rb

```diff
@@ -0,0 +1,399 @@
+require 'helper'
+require 'flipper/adapters/http'
+require 'webmock/rspec'
+require 'flipper/spec/shared_adapter_specs'
+
+RSpec.describe Flipper::Adapters::Http do
```

Comment:
My only concern with this class is that it could easily get out of date with the shared adapter specs. I wonder how hard it would be to mount the API in a rack app, point this test adapter at that and then just use the shared adapter specs with additional specs for the http specific stuff like configuration. 

A quick google brought up [this](http://stackoverflow.com/questions/10166611/launching-a-web-server-inside-ruby-tests) which mentions/links to a few ideas. Something [like this](https://github.com/teamcapybara/capybara/blob/dbeec59726a983b801e56838670d8dd86b6e4f8f/spec/server_spec.rb#L7-L12) would be super rad! What do you think?